README.md

```diff
@@ -103,32 +98,34 @@ print(result.output)
 An agent that has the access to E2B Code Interpreter and is capable of solving complex coding tasks.
 
 ```python
+import os
+
 from dynamiq.nodes.llms.openai import OpenAI
 from dynamiq.connections import OpenAI as OpenAIConnection, E2B as E2BConnection
 from dynamiq.nodes.agents.react import ReActAgent
 from dynamiq.nodes.tools.e2b_sandbox import E2BInterpreterTool
 
 # Initialize the E2B tool
 e2b_tool = E2BInterpreterTool(
-    connection=E2BConnection(api_key="$API_KEY")
+    connection=E2BConnection(api_key=os.getenv("E2B_API_KEY"))
 )
 
 # Setup your LLM
 llm = OpenAI(
     id="openai",
-    connection=OpenAIConnection(api_key="$API_KEY"),
+    connection=OpenAIConnection(api_key=os.getenv("OPENAI_API_KEY")),
     model="gpt-4o",
     temperature=0.3,
     max_tokens=1000,
 )
 
 # Create the ReAct agent
 agent = ReActAgent(
-    name="react-agent",
-    llm=llm,
-    tools=[e2b_tool],
-    role="Senior Data Scientist",
-    max_loops=10,
+    name="react-agent", # Name of the agent
```

Comment:
Do we need to add so obvious comments like name is name, role is role, id is id and so on? Please just add for really no clear attributes.